### DISCUSSION TITLE
is there another repository of the app?

### DISCUSSION THREAD

**Discussion by HeyListenNavi** [z-huang/music#1311] [General]
Hi! i'm not sure if this is the right place to ask this but i recently found another [repository](https://github.com/Arturo254/InnerTune) that claims to be this one? is it related to the app or should i keep using this one? Thanks in advance <3

**Comment by z-huang** [comment]
It looks like a fork of @Malopieds 's fork. However, the developer removed all attribution and didn't use a new app name for his fork. :/

**Comment by z-huang** [comment]
See #1257

**Comment by HeyListenNavi** [comment]
ah i see, it's a shame to hear that :/
thank you anyway!

**Reply by vJoN4** [reply]
@z-huang 

Hi, I have the same concern about this other [repo](https://github.com/Arturo254/InnerTune), specially because in a mexican facebook group called ["Waifus Informaticas"](https://www.facebook.com/groups/517341968854968) this same user is promotting the application in this [post](https://www.facebook.com/groups/517341968854968/posts/1568713693717785) as if were his, describing that your repo is a fork made from a repo of his, but he says that specific repo was deleted due to his original account being suspended.

So, is there any way you can affirm or deny what this other user says?, because he's getting credits from the people in this group.

![image](https://github.com/user-attachments/assets/913a8f96-f97a-4541-b43a-a746d637630a)

**Reply by z-huang** [reply]
> So, is there any way you can affirm or deny what this other user says?, because he's getting credits from the people in this group.

My repo has the commit history from the very first version (0.1.0 beta), a total of thousands of commit. But his repo has only <100 commits. Also, his initial release version is 0.5.3.

I saw that he added attribution to the bottom of his readme hours ago...

**Reply by vJoN4** [reply]
@z-huang 
First of all thanks for answering.

> a total of thousands of commit...

I know, which leads me to think that **your repo is (by far) the original**. But based on what this person is saying, I wanted to ask you directly if there's something else that you can add to deny (or confirm) what he's saying about being the original creator of the app (to notice, I'm still not convinced about his story of your repo being a fork, and I wouldn't like if someone is getting credit from something he didn't do).

> I saw that he added attribution to the bottom of his readme hours ago...

About this addition, I think it's because on the post from the screeenshot I was making him too many questions about how is he supposedly the original owner of the project, and he basically answered that he started the project 5 years ago (around 2019, he didn't give an exact date), and then he 'admited' about being sorry to do not giving you and all the others collaboratos credit for the project. But still, implying that he is the original creator of the app.

If you need screenshots or something about what I was asking him, tell me and I can share them to you translated (because they are in Spanish). I hope you can help me to understand if he's lying or what's going on. 

In advance, thanks!.

**Reply by z-huang** [reply]
Take a look at #35 and #305. People discussed on how to give this app a new name and icon (which is currently being used). I think they are firm evidence about my repo's originality. If he _were_ the original creator, why did he use the same app name as mine?

Second evidence, see the initial commit in his repo https://github.com/Arturo254/InnerTune/commit/96a9552697842b84fe5ff8e82b4df1f3d04b3d8a, it's just a copy from my repo without changing the package name.

**Reply by vJoN4** [reply]
@z-huang 

Thanks again for your answers, I reviewed the issues, and I'm still sure that you are the original creator.

In any case, I'm going to still question this person about his premises in order to find the entire truth and stop any disinformation, because I want this project to get the respect it deserves for all the work and efforts made by you and all the collaborators.

(Specially because there was some people really happy with your app in the post I linked, and they were thinking that this person was you, even someone added an screenshot of the app from a version that includes your name "by Zion Huan", so I didnt like at all the situation).

I'll try to keep you informed about how this goes and what this person answer to my questions.

**Reply by z-huang** [reply]
I opened https://github.com/Arturo254/InnerTune/issues/79 to ask the developer to change his app name and icon to reduce misunderstandings between users.

**Reply by nvllz** [reply]
> In any case, I'm going to still question this person about his premises in order to find the entire truth and stop any disinformation, because I want this project to get the respect it deserves for all the work and efforts made by you and all the collaborators.

@vJoN4 I think you can do better by taking the time to help develop this repo. Fighting with other people who have kept the project alive won't really get you anywhere.
